record_tx_strategy_external_incoming_tx.go

```diff
@@ -27,14 +27,14 @@ func (strategy *externalIncomingTx) Execute(ctx context.Context, c ClientInterfa
 		return nil, fmt.Errorf("ExternalIncomingTx.Execute(): creation of external incoming tx failed. Reason: %w", err)
 	}
 
-	logger.Info(ctx, fmt.Sprintf("ExternalIncomingTx.Execute(): start without ITC, TxID: %s", transaction.ID))
+	logger.Info().Msgf("ExternalIncomingTx.Execute(): start without ITC, TxID: %s", transaction.ID)
```

Comment:
`ExternalIncomingTx.Execute()` Can be added to `Str()`

Reply:
Removed function name from logs